NFIQ2/NFIQ2Algorithm/src/tool/nfiq2_ui_log.cpp

```diff
@@ -46,16 +47,35 @@ void NFIQ2UI::Log::printScore(
   const std::string& name, uint8_t fingerCode, unsigned int score,
   const std::string& errmsg, const bool quantized, const bool resampled,
   const std::list<NFIQ::QualityFeatureData>& featureVector,
-  const std::list<NFIQ::QualityFeatureSpeed>& featureTimings ) const
+  const std::list<NFIQ::QualityFeatureSpeed>& featureTimings,
+  const std::list<NFIQ::ActionableQualityFeedback>& actionableQuality ) const
 {
 
   *( this->out ) << name << "," << std::to_string( fingerCode ) << "," << score
                  << "," << errmsg << "," << quantized << "," << resampled;
-  if( this->verbose || this->speed )
+  if( this->actionable || this->verbose || this->speed )
   {
     *( this->out ) << ",";
   }
 
+  // Print out actionable first
+  if( this->actionable )
+  {
+    for( auto i = actionableQuality.begin(); i != actionableQuality.end(); ++i )
+    {
+      if( i != actionableQuality.begin() )
+      {
+        *( this->out ) << ",";
+      }
+
+      *( this->out ) << std::setprecision( 5 ) << i->actionableQualityValue; // need to check this
```

Comment:
What is needed to be checked with this comment?

Reply:
Removed it. It was a personal comment for me during development.